### DISCUSSION TITLE
word中通过js插入表格，表格首列合并，跨页后表格首列合并状态没有了

### DISCUSSION THREAD

**Discussion by linchong-java** [umodoc/editor#113] [General]
![QQ图片20241107095747](https://github.com/user-attachments/assets/ecdd29a7-5272-481b-bbb2-5f1c796712fb)
![QQ图片20241107095715](https://github.com/user-attachments/assets/0f2dca05-5eb2-457e-8917-4470afacb71a)

大佬们，umodoc文档中，word中通过js插入表格，表格首列合并，跨页后表格首列合并状态没有了

**Comment by umodoc** [comment]
见 https://github.com/umodoc/editor/discussions/153
